SConstruct

```diff
@@ -94,7 +94,7 @@ game_libs = [
 	"png",
 	"jpeg",
 	"openal",
-	"pthread",
+	"pthread"
```

Comment:
minor comment: I think that the trailing comma was a deliberate choice (to allow adding new options without the previous line showing up as changed in the PR diff).
C++ doesn't support such trailing commas on the last entry, but Python does (and SCons is python based).